src/components/Profile/styled.ts

```diff
@@ -100,3 +91,32 @@ export const ProfileButton = styled(Button)`
 		display: block;
 	}
 `
+
+export const Photo = styled.div`
```

Comment:
Combination of nickname and motto is weirdly centered.

![center](https://user-images.githubusercontent.com/76131194/111762153-f74b8a80-88a0-11eb-8219-bbc114e6bb37.png)

Reply:
I don't see a problem🤔 the lines take up as much space as needed and we don't want the name at the very top.

Reply:
This is what I meant. In current case, each gap (margin or padding) is different. size. 

Name is pinned down:
![current one](https://user-images.githubusercontent.com/76131194/111777859-261f2c00-88b4-11eb-82fb-d6ff4caaf4e1.png)

While motto is centered:
![centered](https://user-images.githubusercontent.com/76131194/111777957-4353fa80-88b4-11eb-8b84-bb7eb85a73ba.png)

Resulting in three different gaps:
![každájiná](https://user-images.githubusercontent.com/76131194/111777665-e0626380-88b3-11eb-9deb-30f01c5191e5.png)

While this is what happens if you don't vertically center motto:
![somethinglikethis](https://user-images.githubusercontent.com/76131194/111777670-e1939080-88b3-11eb-9855-aee3f94dbf24.png)

Motto should be three rows of text max (something like that in number of letters), your first example should be forbidden.